### ISSUE TITLE
Failing at install app deps

### ISSUE THREAD

**Issue by akashnimare** [irccloud/irccloud-desktop#53]
I am not able to run it. npm install gives this error -
```
electron-prebuilt@1.3.2 postinstall /Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-prebuilt
node install.js
undefined postinstall /Users/akka/dev/electronapps/irccloud-desktop
install-app-deps

Installing app dependencies for arch x64 to /Users/akka/dev/electronapps/irccloud-desktop/app
npm ERR! git rev-list -n1 saveas: fatal: ambiguous argument 'saveas': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 saveas: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 saveas: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 saveas:
npm ERR! git rev-list -n1 saveas: fatal: ambiguous argument 'saveas': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 saveas: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 saveas: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 saveas:
npm ERR! git rev-list -n1 saveas: fatal: ambiguous argument 'saveas': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 saveas: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 saveas: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 saveas:
npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/akka/.nvm/versions/node/v6.3.1/bin/node" "/Users/akka/.nvm/versions/node/v6.3.1/lib/node_modules/npm/bin/npm-cli.js" "install" "--production"
npm ERR! node v6.3.1
npm ERR! npm  v3.10.3
npm ERR! code 128

npm ERR! Command failed: git rev-list -n1 saveas
npm ERR! fatal: ambiguous argument 'saveas': unknown revision or path not in the working tree.
npm ERR! Use '--' to separate paths from revisions, like this:
npm ERR! 'git <command> [<revision>...] -- [<file>...]'
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/akka/dev/electronapps/irccloud-desktop/app/npm-debug.log
Unhandled rejection Error: /Users/akka/.nvm/versions/node/v6.3.1/bin/node exited with code 1
    at ChildProcess.childProcess.on.code (/Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/util/util.ts:132:14)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:852:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
From previous event:
    at spawn (/Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/util/util.ts:109:9)
    at spawnNpmProduction (/Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/util/util.ts:43:9)
    at Object.installDependencies (/Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/util/util.ts:21:120)
    at /Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/install-app-deps.ts:30:2
    at next (native)
From previous event:
    at tsAwaiter (/Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/util/awaiter.ts:10:47)
    at main (/Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/install-app-deps.ts:23:3)
    at Object.<anonymous> (/Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/install-app-deps.ts:35:7)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
> electron-prebuilt@1.3.2 postinstall /Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-prebuilt
> node install.js


> undefined postinstall /Users/akka/dev/electronapps/irccloud-desktop
> install-app-deps

Installing app dependencies for arch x64 to /Users/akka/dev/electronapps/irccloud-desktop/app

npm ERR! git rev-list -n1 saveas: fatal: ambiguous argument 'saveas': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 saveas: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 saveas: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 saveas:
npm ERR! git rev-list -n1 saveas: fatal: ambiguous argument 'saveas': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 saveas: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 saveas: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 saveas:
npm ERR! git rev-list -n1 saveas: fatal: ambiguous argument 'saveas': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 saveas: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 saveas: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 saveas:
npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/akka/.nvm/versions/node/v6.3.1/bin/node" "/Users/akka/.nvm/versions/node/v6.3.1/lib/node_modules/npm/bin/npm-cli.js" "install" "--production"
npm ERR! node v6.3.1
npm ERR! npm  v3.10.3
npm ERR! code 128

npm ERR! Command failed: git rev-list -n1 saveas
npm ERR! fatal: ambiguous argument 'saveas': unknown revision or path not in the working tree.
npm ERR! Use '--' to separate paths from revisions, like this:
npm ERR! 'git <command> [<revision>...] -- [<file>...]'
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/akka/dev/electronapps/irccloud-desktop/app/npm-debug.log
Unhandled rejection Error: /Users/akka/.nvm/versions/node/v6.3.1/bin/node exited with code 1
    at ChildProcess.childProcess.on.code (/Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/util/util.ts:132:14)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:852:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
From previous event:
    at spawn (/Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/util/util.ts:109:9)
    at spawnNpmProduction (/Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/util/util.ts:43:9)
    at Object.installDependencies (/Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/util/util.ts:21:120)
    at /Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/install-app-deps.ts:30:2
    at next (native)
From previous event:
    at tsAwaiter (/Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/util/awaiter.ts:10:47)
    at main (/Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/install-app-deps.ts:23:3)
    at Object.<anonymous> (/Users/akka/dev/electronapps/irccloud-desktop/node_modules/electron-builder/src/install-app-deps.ts:35:7)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
```

**Comment by jwheare** [comment]
This should be fixed by fafe399 please try again.

**Comment by akashnimare** [comment]
@jwheare fixed 😄
